core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -318,6 +319,30 @@ private function addSecurityNode(): NodeDefinition
         ;
     }
 
+    private function addSearchNode(): NodeDefinition
+    {
+        return (new TreeBuilder('search'))
+            ->getRootNode()
+            ->addDefaultsIfNotSet()
+            ->children()
+                ->arrayNode('default_indexer')
+                    ->info('Settings regarding the default search indexer which indexes pages in the database.')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('enable')
+                            ->info('Enables the default search indexer')
+                            ->defaultTrue()
+                        ->end()
+                        ->scalarNode('enableIndexProtected')
```

Comment:
```suggestion
                        ->scalarNode('indexProtected')
```

Reply:
Right. I've also noticed that this is a rather general setting and isn't related to our default indexer at all. If I'm adding another indexer I still might want to index protected pages or not. So I've moved that to `search.indexProtected` directly in 29d61964184d9384111f467f69121097b0dced5e.